jax_unirep/evotuning.py

```diff
@@ -136,11 +157,32 @@ def length_batch_input_outputs(
     return xs, ys
 
 
+def get_batch_len(batched_seqs):
+    """
+    Returns the average length of each batch as well as a full array of batch distribution.
+
+    :param batched_seqs: list of lists of sequences, grouped by length.
+    """
+    batch_lens = []
+    for batch in batched_seqs:
+        batch_lens.append(len(batch))
+
+    batch_lens = np.array(batch_lens)
+
+    return np.mean(batch_lens), batch_lens
+
+
 def fit(
-    params: Dict, sequences: List[str], n: int, step_size: float = 0.001
+    params: Dict,
+    sequences: List[str],
+    n: int,
+    step_size: float = 0.001,
+    holdout_seqs: Optional[List[str]] = None,
+    proj_name: Optional[str] = "temp",
+    steps_per_print: Optional[int] = 200,
 ) -> Dict:
     """
-    Return weights fitted to predict the next letter in each sequence.
+    Returns weights fitted to predict the next letter in each sequence.
```

Comment:
```suggestion
    Return weights fitted to predict the next letter in each sequence.
```

Pedantic little nitpick from me - IIRC, it should be “Return” and not “Returns”. There’s a principle to phrasing the first line of docstrings that I can’t remember at the moment.